src/components/SceneGraph.js

```diff
@@ -176,8 +178,15 @@ export default class SceneGraph extends React.Component {
         let className = 'option' + (option.value === this.state.value ? ' active' : '');
         return (
           <div key={idx} className={className} value={option.value}
-            dangerouslySetInnerHTML={{__html: option.html}}
-            onClick={() => {this.handleEntityClick(option.value)}}/>
+            onClick={this.setValue.bind(this, option.value)}>
```

Comment:
done! I've moved also the tracking's `ga` code from `handleEntityClick` to `setValue`